Sources/TuistGenerator/Generator/ConfigGenerator.swift

```diff
@@ -148,6 +149,21 @@ final class ConfigGenerator: ConfigGenerating {
         pbxproj.add(object: variantBuildConfiguration)
         configurationList.buildConfigurations.append(variantBuildConfiguration)
     }
+    
+    private func update(buildSettings settings: inout [String: SettingValue], byApplying manifest: Target) {
+        if let deploymentTarget = manifest.deploymentTarget {
+            switch deploymentTarget {
+            case let .iOS(_, devices):
+                if manifest.platform == .iOS {
+                    settings["TARGETED_DEVICE_FAMILY"] = .string(devices.map { "\($0.rawValue)" }.joined(separator: ","))
+                }
+            case .macOS:
+                if manifest.platform == .macOS {
+                    settings["TARGETED_DEVICE_FAMILY"] = .string("\(DeploymentDevice.mac.rawValue)")
```

Comment:
I had a look at `BuildSettingsProvider` from `XcodeProj` project, and it looks like we do not set `TARGETED_DEVICE_FAMILY` for `macOS` platform there 🤔I guess we should not set `TARGETED_DEVICE_FAMILY` for `macOS` in `ConfigGenerator` either, right?